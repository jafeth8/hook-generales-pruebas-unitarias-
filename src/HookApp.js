export const HookApp=()=>{
    return (
        <>
            <h1>hola mundo</h1>
        </>
    )
}