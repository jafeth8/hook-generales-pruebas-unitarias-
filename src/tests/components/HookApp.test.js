import "@testing-library/jest-dom";
import {shallow} from 'enzyme';
import { HookApp } from "../../HookApp";

describe('HookApp component', () => {
    test('mostrar correctamente el componente', () => {
        const wrapper = shallow(<HookApp/>)
        expect(wrapper).toMatchSnapshot();
    })
    
})
