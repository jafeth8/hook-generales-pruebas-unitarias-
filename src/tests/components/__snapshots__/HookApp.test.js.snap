// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HookApp component mostrar correctamente el componente 1`] = `
<Fragment>
  <h1>
    hola mundo
  </h1>
</Fragment>
`;
